mscore/qml/palettes/PalettesWidgetHeader.qml

```diff
@@ -127,6 +127,9 @@ Item {
 
     Connections {
         target: mscore
-        onPaletteSearchRequested: searchTextInput.forceActiveFocus()
+        onPaletteSearchRequested: {
+            searchTextInput.forceActiveFocus()
+            searchTextInput.selectAll()
+        }
     }
```

Comment:
Didn't use banner-style brace-indention because the current QML code here is following the standard C++ style.

Reply:
Yes